### DISCUSSION TITLE
Lucee 6 support

### DISCUSSION THREAD

**Discussion by guustnieuwenhuis** [MasaCMS/MasaCMS#273] [General]
Hi all,

We've working on support for Lucee 6. 

In our latest release (7.4.5), we've already fixed some issue. But we've also noticed a new issue in the latest Lucee 6 beta release (6.0.0.451-BETA) that didn't show up in the previous beta release (6.0.0.346-BETA).

**Comment by mcaresimo1970** [comment]
Hi All
Has this now been addressed? We are currently on Lucee 5.3.10.120 and were wondering what the latest version of Lucee we can update to that works with 7.5.1
Many thanks

**Reply by Tigerlady** [reply]
We are currently running on Lucee 6.0.4.10 and are running the lateset version of MASA.  We plan on doing an update this weekend to the next version on our Dev instance of lucee and then testing.  If all goes well we will increase the versions till we are on the latest version of Lucee that is in production. Hope that helps.

**Reply by mcaresimo1970** [reply]
Super. Many thanks. Much appreciated!

**Reply by Tigerlady** [reply]
No problem.  We always have a dev instance of lucee that we put dev instances of website on before we do any live instance updates.  It helps us to identify any issues before causing problems with live sites.  We do hosting for various developers and for every live site we also set up a dev site for testing.  In case you are in need of a hosting company.

**Reply by guustnieuwenhuis** [reply]
Masa CMS runs fine on Lucee 6, but you will need Masa CMS 7.5.4 or higher.
